cert_issuer/blockchain_handlers/ethereum/connectors.py

```diff
@@ -135,7 +135,7 @@ def get_address_nonce(self, address, api_token):
         """
         broadcast_url = self.base_url + '?module=proxy&action=eth_getTransactionCount'
         broadcast_url += '&address=%s' % address
-        broadcast_url += '&tag=latest'
+        broadcast_url += '&tag=pending' # Valid tags are 'earliest', 'latest', and 'pending', the last of which includes both pending and committed transactions.
```

Comment:
I can't tag this far down but there's other `get_address_nonce` pulling from `latest` in this file.

Reply:
Fixed this, thanks for catching it.